web/src/main.tsx

```diff
@@ -72,8 +73,8 @@ const router = createBrowserRouter([
     ),
   },
   {
-    path: '/schemas',
-    element: <Schemas />,
+    path: '/browse',
```

Comment:
We could keep schemas, and then just reroute them to `/browse`

Reply:
fixed